### DISCUSSION TITLE
2024-09-06 온라인 회의

### DISCUSSION THREAD

**Discussion by nyj001012** [jiphyeonjeon-42/backend-nest#28] [General]
참여자 : jimin, yena, minsepar, youkim

지난 시간 목표설정 중간점검
이슈 공유

**Comment by nyj001012** [comment]
## 진행도
### 기존 api 하나 service, repository, swagger까지 nestjs로 변환하여 PR 올리기
- controller, service `histories` 보일러 플레이트 작성
- TypeORM 연동 시도 중
- 리포지토리 패턴 고안 중

## 목표
- TypeORM 연동

**Comment by parkj12b** [comment]
## 진행도
### 아무것도 못함 ㅠㅠ 

## 목표
- v1 user nest 로 옮기는거 진행하기

**Comment by scarf005** [comment]
## 진행도

새로 들어왔습니다 반갑습니다~

## 목표

- nestjs 입문
- 주말동안 nestjs swagger 연동

**Comment by scarf005** [comment]
## 명세가 필요할 것 같습니다

- 명세가 있으면 의사소통 비용이 줄고 예상치 못한 개발을 줄일 수 있습니다.
- 서비스를 통해 무엇을 할 수 있는지 정리하면 좋을 것 같습니다.
	- 예시: 도서는 몇 권을 빌릴 수 있으며, 대출 기간은 몇 일, 연체시 동작 등...

### 참고

- https://mermaid.js.org/syntax/sequenceDiagram.html
- https://mermaid.js.org/syntax/flowchart.html
